src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1322,6 +1322,7 @@
     <Field Name="Expanded" Type="bool"/>
     <Field Name="ArgsToParamsOpt" Type="ImmutableArray&lt;int&gt;" Null="allow"/>
     <Field Name="ResultKind" PropertyOverrides="true" Type="LookupResultKind"/>
+    <Field Name="InstanceSymbol" Type="TypeSymbol" Null="disallow"/>
```

Comment:
The term `Instance` is not used elsewhere in `BoundNodes`. Perhaps rename to `ReceiverType`.

Reply:
Updated.